.github/workflows/ci.yml

```diff
@@ -140,3 +140,31 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         args: --workspace --all-features --all-targets -- -D warnings
+  minimal_versions:
+    name: Minimal versions
+    needs: smoke
+    strategy:
+      matrix:
+        os: [ "ubuntu-latest", "windows-latest", "macos-latest" ]
```

Comment:
Like with `check` vs `test`, do we need all platforms or just one?

Reply:
> Yeah, but there is still that 1% when your minimal crate version is some bugfix, that is tested inside the crate.

Theoretically in this 1% maybe itself 1% of cases where this bugfix is OS-dependent